uint8 ctrl_mode      #  0：位置  1：速度  2：力矩
float32 position     # 位置
float32 velocity     # 速度
float32 torque       # 力矩